src/common/meta/src/error.rs

```diff
@@ -296,6 +296,37 @@ pub enum Error {
         error: serde_json::Error,
         location: Location,
     },
+
+    #[snafu(display("Invalid number of topics {}", num_topics))]
+    InvalidNumTopics {
+        num_topics: usize,
+        location: Location,
+    },
+
+    #[snafu(display(
+        "Failed to build an rskafka client, broker endpoints: {:?}",
+        broker_endpoints
+    ))]
+    BuildClient {
```

Comment:
```suggestion
    BuildKafkaClient {
```

Reply:
We may have other clients in the future, so the name...

Reply:
Updated.